resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -813,78 +949,22 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                 }
 
                 final DnsQueryContext qCtx = promises.get(queryId);
-
                 if (qCtx == null) {
                     if (logger.isWarnEnabled()) {
-                        logger.warn("Received a DNS response with an unknown ID: {}", queryId);
+                        logger.warn("{} Received a DNS response with an unknown ID: {}", ch, queryId);
```

Comment:
I think it has to be `Received`, because it's the first word in the line.

```
[0x123abc] Received a DNS response ...
```
vs.
```
[0x123abc] received a DNS response ...
```

We would need to use `received` if it were an exception message:

```
io.netty.foo.BarException: received a DNS response ...
```
vs.
```
io.netty.foo.BarException: Received a DNS response ...
```

Well, maybe it's confusing enough and we should just always start with a capitalized word unless it's a variable name.